docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -206,10 +206,7 @@ If you need to use an integration that you have previously revoked, log in again
 
 ## Why does Codacy ask for permission to create SSH keys?
 
-Codacy asks for permission to create SSH keys because it needs to create an SSH key in your account in the following situations:
-
--   If your repository uses submodules, so that Codacy can clone the repositories for each submodule
--   If Codacy fails to integrate with a repository using the repository key, so that Codacy can continue to perform analysis
+Codacy asks for permission to create SSH keys because it needs to create an SSH key in your account so that it can also clone the repositories for each submodule, if your repository uses git submodules.
```

Comment:
I suggest simplifying a bit to remove redundancy and clarify that we only use SSH keys to clone submodules (that "also" sounds a bit suspicious):

```suggestion
Codacy needs to create an SSH key in your account to clone the repositories for each submodule if your repository uses git submodules.
```

Reply:
hmmm, we use the same SSH key to clone both the top level repository, and any submodules it might have